src/psl.c

```diff
@@ -49,6 +49,7 @@
 
 #ifdef _WIN32
 /* This is for Windows NT/2000 and later, for WSAStringToAddressW() */
+# undef _WIN32_WINNT
```

Comment:
This throws a warning with some compilers (e.g. clang) if _WIN32_WINNT is not defined. Since these are more often used than MSVC, please remove this commit.

Reply:
I added this line becase I'm getting this warning without it:

```
  CC       libpsl_la-psl.lo
../../src/psl.c:53:0: warning: "_WIN32_WINNT" redefined
 # define _WIN32_WINNT 0x0500

In file included from /usr/i686-w64-mingw32/sys-root/mingw/include/crtdefs.h:10:0,
                 from /usr/i686-w64-mingw32/sys-root/mingw/include/sys/types.h:13,
                 from ../../src/psl.c:47:
/usr/i686-w64-mingw32/sys-root/mingw/include/_mingw.h:232:0: note: this is the location of the previous definition
 #define _WIN32_WINNT 0x502
```

It is not a big problem however, if you prefer I can remove it.